src/containers/Guide/GuideIntroduction/index.tsx

```diff
@@ -38,18 +38,52 @@ const GuideIntroduction: FC = () => {
+        </p>
+        <DocSubHeader>Incentives</DocSubHeader>
+        <p>
+          This leaves us with the challenge of humans themselves. Humans, unlike computer algorithms, can be corrupted.
+          However, because of how we want to reward our community members, we believe that there is no better system
+          than human judgement for quantifying a wide variety of work (knowing that many jobs are still unknown).
+          Choosing a government who truly has the best interest of the community at heart can ensure prosperity, as it
+          happens in actual life, in enterprises, institutions, organizations, or countries around the globe. No matter
+          how many checks and balances we put in place, no matter how many restrictions or regulations we write, we will
+          be always getting back to the variable of human judgement.
+        </p>
+        <DocSubHeader>Value and work put into the community</DocSubHeader>
+        <p>
+          We want to ensure that everyone (not just developers and DevOps) can receive the value of their work put into
+          making our network and community better. In the long run, this is the only way to guarantee success for our
+          community as a group of people working toward common goals. In fact, we see thenewboston as an everlasting
+          project because only work put into it generates value spread across every unit of our cryptocurrency. Every
+          unit of thenewboston will forever represent real human effort put into it, and in every trade of thenewboston
+          this human effort will add to the cryptocurrency's value. This also demystifies how every single coin becomes
+          valuable at the moment we generate it.
+        </p>
+        <DocSubHeader>Distribution model deriving from the real world</DocSubHeader>
+        <p>
+          We do not base our distribution model on Proof of Work (PoW) or Proof of Stake (PoS) like most other
+          cryptocurrencies. With Bitcoin's PoW, for example, the monetary incentive ties to mining blocks, which results
+          in significant loss of time and energy attempting to mine blocks faster, rather than dealing with actual
+          problems that must be solved. This includes building mobile apps, developing payment libraries, design,
+          marketing, research, and so on.
+        </p>
+        <DocSubHeader>Efficiency</DocSubHeader>
+        <p>
+          When building a distributed payment ledger, it is not the transaction processing itself that requires
+          distribution across multiple servers. This often results in duplicate work being done by several network
+          nodes, causing an inherent inefficiency in the system. It is rather the ability to elect fairly a single
+          validation server and consensual acceptance of the produced results that require distribution among peers.
+          This allows for highly performant transaction validation within a decentralized network.
```

Comment:
This could be clearer - 

maybe comment the intent of this section and perhaps i can help in communicating the message.

Reply:
It is the original paragraph I believe.